modules/nf-core/nanoplot/main.nf

```diff
@@ -2,10 +2,10 @@ process NANOPLOT {
     tag "$meta.id"
     label 'process_low'
 
-    conda "bioconda::nanoplot=1.40.0"
+    conda "bioconda::nanoplot=1.41.0"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/nanoplot:1.40.0--pyhdfd78af_0' :
-        'quay.io/biocontainers/nanoplot:1.40.0--pyhdfd78af_0' }"
+        'https://depot.galaxyproject.org/singularity/nanoplot:1.41.0--pyhdfd78af_0' :
+        'quay.io/biocontainers/nanoplot:1.41.0--pyhdfd78af_0' }"
```

Comment:
Can this be in a separate PR?

Reply:
This change is actually already on master: https://github.com/nf-core/modules/blob/master/modules/nf-core/nanoplot/main.nf#L5-L8
I don't know why GitHub is showing it in this PR. I noticed it's been doing that lately, and it's confusing

Reply:
I pulled the latest version of the remote to "master" branch and confirmed with `git diff master HEAD modules/nf-core/nanoplot/main.nf` and `git checkout master modules/nf-core/nanoplot/main.nf` that there are no changes. These files are identical to those already on master.